src/node.jl

```diff
@@ -77,12 +77,24 @@ function Bonobo.get_branching_nodes_info(tree::Bonobo.BnBTree, node::FrankWolfeN
 
     # In case of strong convexity, check if a child can be pruned
     prune_left, prune_right = prune_children(tree, node, lower_bound_base, x, vidx)
+    #================================================================================#
+    #different ways to split active set
+    if !(tree.root.options[:variant] == DICG() || tree.root.options[:variant] == VFW() )
```

Comment:
The splitting of the active set is also necessary if the variant is `VanillaFrankWolfe`.